src/layup/utilities/file_io/file_output.py

```diff
@@ -14,8 +14,19 @@ def write_csv(data, filepath):
         The data to write to the file.
     filepath : str
         The path to the file to write.
+    move_columns : dict, optional
+        Dict of any column names that need moved, paired with their new position.
     """
     df = pd.DataFrame(data)
+
+    if move_columns != {}:
+        column_names = list(df.columns.values)
```

Comment:
what happens if the user sent a column name that doesn't exist in the data array? Need to check and error on that. you might be able to do a try and except

https://www.w3schools.com/python/python_try_except.asp